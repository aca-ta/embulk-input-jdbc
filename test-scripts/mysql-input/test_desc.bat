mysql -uTEST_USER -pXXXXXXXX -DTESTDB < test.sql

del data\test000.00.csv
CALL embulk run test_desc.yml

echo "diff data/test_desc_expected.csv data/test000.00.csv"
diff data/test_desc_expected.csv data/test000.00.csv

IF "%ERRORLEVEL%" == "0" (ECHO "OK!") ELSE (ECHO "embulk-input-mysql desc FAILED!")
