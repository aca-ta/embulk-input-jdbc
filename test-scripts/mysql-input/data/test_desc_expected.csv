ID,NUM,DT
3,,
2,1234567890.12,2015-12-31
1,123.40,2015-04-24
